backend/postgres/postgres.go

```diff
@@ -137,7 +137,6 @@ func init() {
 	if cfg.ApplicationName == "-" {
 		names := []string{
 			os.Getenv("JAEGER_SERVICE_NAME"),
-			os.Getenv("SERVICE_NAME"),
```

Comment:
This is safe to remove, because we never actually used it?

Reply:
This overall logic is highly confusing. I would rather keep the `SERVICE_NAME ` and in case, none is given log a warning that the `JAEGER_SERVICE_NAME` was used as fallback option to make the behaviour more transparent to the user.